broadinstitute/single_cell_portal_core
# Add the badges for TDR results for facet matching (SCP-3473)

This adds the facet matching badge for TDR results in the study search.

To Test locally:
Try searching for a study using the facet "homo sapien" and you should see the TDR result with the badge same as any SCP study 

Search on both keyword and facet matching:
<img width="1191" alt="Screen Shot 2021-07-30 at 12 30 49 PM" src="https://user-images.githubusercontent.com/54322292/127872863-9fd43d1e-c04b-4c50-afa9-4ab6c1a6f719.png">

Hovering tooltip on TDR:
<img width="311" alt="Screen Shot 2021-07-30 at 12 30 56 PM" src="https://user-images.githubusercontent.com/54322292/127872864-8351fed7-0ed3-461f-ba81-751b8039c400.png">

Just Facet match:
<img width="804" alt="Screen Shot 2021-07-30 at 12 31 08 PM" src="https://user-images.githubusercontent.com/54322292/127872865-5d16b03d-8810-4ee9-9428-25804d3e019f.png">

Just keyword match still works as usual
<img width="911" alt="Screen Shot 2021-07-30 at 12 31 22 PM" src="https://user-images.githubusercontent.com/54322292/127872866-4935f6a0-7ea6-46a6-a089-80d2d6a2dd90.png">
